src/wasm-traversal.h

```diff
@@ -208,8 +224,8 @@ struct Walker : public VisitorType {
     for (auto& curr : module->functions) {
       self->walkFunction(curr.get());
     }
-    self->visitTable(&module->table);
-    self->visitMemory(&module->memory);
+    self->walkTable(&module->table);
```

Comment:
I don't have that much context around this but it seems straightforward.
Making sure I understand: before we would defer to how the SubType `visit`ed the table/memory, which didn't necessarily `walk` the entire module. Walks being recursive, visits being non-recursive?
If so this seems correct.